csv_to_openfisca.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+
+import pandas as pds
+import numpy as np
+
+# from openfisca_core.simulations import Simulation
+from openfisca_core.simulation_builder import SimulationBuilder, Simulation
+from openfisca_country_template import CountryTaxBenefitSystem
+
+tax_benefit_system = CountryTaxBenefitSystem()
+
+def set_members_entity_id(entity, entity_id_array):
+  assert entity.ids is not None  # We needs the ids to be set to make sure we match households.
+  ids, inverse = entity.members_entity_id = np.unique(entity_id_array, return_inverse = True)
+  assert (ids == np.unique(entity.ids)).all()  # otherwise, input are inconsistent
+  entity.members_entity_id = np.argsort(entity.ids)[inverse]
+
+
+def set_members_role(entity, role_array):
+  # We should check that there is not an unexpected role here.
+  entity.members_role = np.select([role_array == role.key for role in entity.flattened_roles], entity.flattened_roles)
```

Comment:
This is assuming roles will be provided by their string id.

Reply:
FI,  right now the Survey Manager doesn't do that and use a "legacy role" vector, which has int values. (0 means first parent, 1 second parent, 2 first children, 3 second children, etc.)